src/transport.c

```diff
@@ -108,10 +108,13 @@ static int find_address(unsigned char ipaddr[BOOTH_IPADDR_LEN],
 				break;
 
 		if (matched == node->addrlen) {
-			*address_bits_matched = matched * 8;
-			*me = node;
-			did_match = EXACT_MATCH;
-			break;
+			if((matched_tmp < matched * 8)||((node->type == SITE)&&(matched_tmp == matched * 8)))
```

Comment:
The travis build failed. Anyway, I'm not sure if I understand the problem here. If a request for the ticket is received at the booth member which is not the ticket leader, that request is going to be forwarded to the leader. @liu4480, did you observe this actually happening? Can you please post your configuration and exact circumstances.